client/src/components/atoms/SubmissionToolbar.vue

```diff
@@ -32,84 +32,95 @@
           </q-chip>
         </div>
       </q-toolbar-title>
-      <div v-if="submission.status == 'DRAFT'">
-        <q-btn
-          data-cy="initially_submit"
-          rounded
-          color="positive"
-          :label="$t(`submission.action.submit_for_review`)"
-          class="q-ml-md"
-          @click="confirmHandler('submit_for_review')"
-        >
-        </q-btn>
-      </div>
-      <div
-        v-else-if="
-          submission.status != 'AWAITING_REVIEW' &&
-          submission.status != 'REJECTED' &&
-          submission.status != 'RESUBMISSION_REQUESTED'
-        "
-        data-cy="decision_options"
+
+      <q-btn-dropdown
+        label="Options"
+        flat
+        menu-anchor="bottom right"
+        menu-self="top right"
       >
-        <q-btn
-          v-if="submission.status == 'INITIALLY_SUBMITTED'"
-          data-cy="accept_for_review"
-          rounded
-          color="positive"
-          :label="$t(`submission.action.accept_for_review`)"
-          class="q-ml-md"
-          @click="confirmHandler('accept_for_review')"
+        <div v-if="submission.status == 'DRAFT'">
+          <q-btn
+            data-cy="initially_submit"
+            rounded
+            color="positive"
+            :label="$t(`submission.action.submit_for_review`)"
+            class="q-ml-md"
+            @click="confirmHandler('submit_for_review')"
+          >
+          </q-btn>
+        </div>
+        <q-btn-group
+          flat
+          square
+          v-else-if="
+            submission.status != 'AWAITING_REVIEW' &&
+            submission.status != 'REJECTED' &&
+            submission.status != 'RESUBMISSION_REQUESTED'
+          "
+          data-cy="decision_options"
+          class="column"
```

Comment:
```suggestion
          class="column q-pa-sm"
```
This will make it easier to distinguish this button group dropdown from the content it appears on top of.